### ISSUE TITLE
AirflowNetwork objects not used without AFN:SimControl object and no error/warning message

### ISSUE THREAD

**Issue by matthew-larson** [NREL/EnergyPlus#8010]
Issue overview
--------------
Any model with AFN objects but no AirflowNetwork:SimulationControl object simulates successfully without using any of the AFN objects (no AFN-related output variables shown in RDD) and EnergyPlus doesn't add any warning or error messages to the error file. This is the intended behavior, but should have at least a warning generated that lets the user know their AFN objects won't be used.  I could see new users falling into a trap where they didn't add the AirflowNetwork:SimulationControl object and thought they set up all other AFN objects correctly because there are no related warnings or errors.

### Details
Some additional details for this issue (if relevant):
 - Version 9.3

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jasondegraw** [comment]
I don't entirely agree that this is necessarily needed, I thought the documentation was clear on needing the simulation control object. I'll go double check. I don't agree that this is a defect since it is the intended behavior.

**Comment by nealkruis** [comment]
@jasondegraw I think you're right that this is covered in the documentation. I would argue that the intended behavior is defective and confusing for users. Why not add a warning to point a user in the right direction if they are confused?

**Comment by jasondegraw** [comment]
@nealkruis I'm not entirely opposed to the idea. It'll need to be compatible with the way that unused objects are handled elsewhere, so without looking at that I'm not sure I can judge complexity. If it can be done in a good way that's helpful, sure, let's do it. But it is not a defect. It's fine to make that argument if you want to (I wouldn't), but:
* it is the current behavior, and
* it is the intended behavior.

That makes this a new feature request.

**Comment by lgu1234** [comment]
@jasondegraw @nealkruis I agree with Jason's concern. This issue is a new feature request. We should treat the AFN objects as unused objects.

**Comment by mjwitte** [comment]
With all due respect, this can be quite frustrating for users (and those answering user support questions). There is precedent for generating a specific error message for this type of situation:
```
   ** Severe  ** Orphaned ZoneHVAC object found.  This was object never referenced in the input, and was not used.
   **   ~~~   **  -- Object type: ZoneHVAC:Baseboard:Convective:Electric
   **   ~~~   **  -- Object name: Unused baseboard
   ************* There are 2 unused objects in input.
   ************* Use Output:Diagnostics,DisplayUnusedObjects; to see them.
```

That said, unused AFN (and any other) objects do generate a subtle message at the end of the error file:
```
   ************* Simulation Error Summary *************
   ************* There are 1 unused objects in input.
   ************* Use Output:Diagnostics,DisplayUnusedObjects; to see them.
```
I don't recall why this is just a message and not a warning, probably due to warning count aversion in code-compliance submittals. 

Interesting, with `Output:Diagnostics,DisplayUnusedObjects;`, it does become a warning:
```
   ** Warning ** The following lines are "Unused Objects".  These objects are in the input
   **   ~~~   **  file but are never obtained by the simulation and therefore are NOT used.
   **   ~~~   **  Only the first unused named object of an object class is shown.  Use Output:Diagnostics,DisplayAllWarnings; to see all.
   **   ~~~   **  See InputOutputReference document for more details.
   ************* Object=AirflowNetwork:MultiZone:Zone=AirflowNetwork:MultiZone:Zone 1
```

1. Perhaps the best way forward here is to elevate the basic unused objects message to a Warning. This requires no change in AFN, just a change to [`InputProcessor::reportOrphanRecordObjects`](https://github.com/NREL/EnergyPlus/blob/fa9c482c780a2954c8bdd3c5480d29f845edcb05/src/EnergyPlus/InputProcessing/InputProcessor.cc#L1328-L1332)

2. And I see that the specific error about unused ZoneHVAC:* equipment is generated in the [same function](https://github.com/NREL/EnergyPlus/blob/fa9c482c780a2954c8bdd3c5480d29f845edcb05/src/EnergyPlus/InputProcessing/InputProcessor.cc#L1292-L1298). So, one could easily add a specific warning about unused AFN objects with a few lines of code in the same place, if desired.

**Comment by jasondegraw** [comment]
@mjwitte Either solution would be fine with me, though I have wondered whether we should just default a simulation control object. If we're going with a message/warning, it would be worth it to include that the input has not been validated. I recall (perhaps incorrectly) that the warning count is no longer a thing. If that's the case, then perhaps it matters less whether it's a message or a warning.

**Comment by matthew-larson** [comment]
Closed with #8454.
